src/v2/client/startLoginFlow.js

```diff
@@ -10,6 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+// eslint-disable-next-line no-redeclare
```

Comment:
I got `'Promise' is already defined as a built-in global variable  no-redeclare`
Looks like we can just remove `global Promise` and problem solved.